Flabbergasted-Flamingos/Bangazon-admin-app
# Rb employee list

# Description
This pull request is for the list of emplyees

## Number of Fixes
#1


## Problem to Solve
This pull request solves the ability for HR to look at employees

## Proposed Changes
The pull request affects views, templates, urls and tests

## Expected Behavior
This is a view only, the hyper links will eventually go to employee detail

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_employee_list
python manage.py runserver
```
2. Make sure employee list populates with department name
3. Run tests
```
python manage.py test
```
Make sure all the tests pass
## Testing

[x ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
I need to make one more test
